Makefile.docker

```diff
@@ -51,13 +36,12 @@ docker_push_default: docker_tag
 .PHONY: docker_save_default
 docker_save_default:
 	# Saves the container as TGZ file
-	test -d $(ARCHIVE_DIR) || mkdir -p  $(ARCHIVE_DIR)
-	docker save strimzi/$(PROJECT_NAME):$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX) | gzip > $(ARCHIVE_DIR)/$(PROJECT_NAME)-$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX).tar.gz
+	docker save strimzi/$(PROJECT_NAME):$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX) | gzip > $(PROJECT_NAME)-$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX).tar.gz
 
 .PHONY: docker_load_default
 docker_load_default:
 	# Loads the container as TGZ file
-	docker load < $(ARCHIVE_DIR)/$(PROJECT_NAME)-$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX).tar.gz
+	docker load < $(PROJECT_NAME)-$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX).tar.gz
```

Comment:
Looks like you removed the archive dir here but not in the pipelines on Azure?

Reply:
Good spot, I've made an update and it's now passing